src/libmongoc/tests/test-mongoc-scram.c

```diff
@@ -669,6 +685,49 @@ test_mongoc_saslprep_auth_no_icu (void *ctx)
    _drop_saslprep_users ();
 }
 
+static void
+test_mongoc_utf8_char_length (void)
+{
+   const char *c0 = ",";
+   const char *c1 = "ɶ";
+   const char *c2 = "ྡྷ";
+   const char *c3 = "🌂";
+
+   ASSERT_CMPINT (_mongoc_utf8_char_length (c0), ==, 1);
+   ASSERT_CMPINT (_mongoc_utf8_char_length (c1), ==, 2);
+   ASSERT_CMPINT (_mongoc_utf8_char_length (c2), ==, 3);
+   ASSERT_CMPINT (_mongoc_utf8_char_length (c3), ==, 4);
+}
+
+static void
+test_mongoc_utf8_string_length (void)
+{
+   const char *s0 = ",ase";
+   const char *s1 = "ɸɴ";
+   const char *s2 = "ྡྷ🌂e4🌕";
+   const char *s3 = "no special characters";
+
+   ASSERT_CMPINT (_mongoc_utf8_string_length (s0), ==, 4);
```

Comment:
```suggestion
   ASSERT_CMPINT (_mongoc_utf8_string_length (",ase"), ==, 4);
```
Local variables may not be necessary for simple, single-use cases like this.